server/store/apps.go

```diff
@@ -12,23 +12,33 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/mattermost/mattermost-plugin-apps/apps"
-	"github.com/mattermost/mattermost-plugin-apps/server/api"
+	"github.com/mattermost/mattermost-plugin-apps/server/config"
 	"github.com/mattermost/mattermost-plugin-apps/server/utils"
 )
 
+type App interface {
+	config.Configurable
+
+	AsMap() map[apps.AppID]*apps.App
+	Delete(apps.AppID) error
+	Get(appID apps.AppID) (*apps.App, error)
+	InitBuiltin(...*apps.App)
+	Save(app *apps.App) error
+}
+
 // appStore combines installed and builtin Apps.  The installed Apps are stored
 // in KV store, and the list of their keys is stored in the config, as a map of
 // AppID->sha1(App).
 type appStore struct {
-	*Store
+	*Service
```

Comment:
1/5 - find more readable `Store` than `Service` in a storage context ( we use `Service all across now and it becomes harder to follow).